examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb

```diff
@@ -206,6 +272,19 @@
     "NOTE: This currently does not work with streaming `ChatCompletion`, but we hope to support it soon!"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -qqq weave tiktoken"
```

Comment:
minor/confusing: I get a message about errors in the installation/needing to restart the runtime, but the rest of the colab runs fine:
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pandas-gbq 0.17.9 requires pyarrow<10.0dev,>=3.0.0, but you have pyarrow 13.0.0 which is incompatible.
tensorflow 2.13.0 requires typing-extensions<4.6.0,>=3.6.6, but you have typing-extensions 4.8.0 which is incompatible.
Successfully installed GitPython-3.1.37 aiofiles-23.2.1 aioprocessing-2.0.1 analytics-python-1.2.9 backoff-2.2.1 black-23.9.1 docker-pycreds-0.4.0 flask-cors-4.0.0 gitdb-4.0.10 gql-3.4.1 graphql-core-3.2.3 ipynbname-2023.2.0.0 janus-1.0.0 jedi-0.19.0 mypy-extensions-1.0.0 objgraph-3.6.0 pathspec-0.11.2 pathtools-0.1.2 pyarrow-13.0.0 pynndescent-0.5.10 python-json-logger-2.0.7 requests-toolbelt-0.10.1 sentry-sdk-1.28.1 setproctitle-1.3.2 smmap-5.0.1 tiktoken-0.5.1 typeguard-4.1.5 typing-extensions-4.8.0 umap-learn-0.5.4 urllib3-1.26.16 wandb-0.15.11 weave-0.28.0
WARNING: The following packages were previously imported in this runtime:
  [urllib3]
You must restart the runtime in order to use newly installed versions.

Reply:
fixed in next commit